subnet/subnet.go

```diff
@@ -148,5 +148,6 @@ type Manager interface {
 	WatchLease(ctx context.Context, sn ip.IP4Net, sn6 ip.IP6Net, cursor interface{}) (LeaseWatchResult, error)
 	WatchLeases(ctx context.Context, cursor interface{}) (LeaseWatchResult, error)
 
+	SetNodeNetworkUnavailableFalse(ctx context.Context) error
```

Comment:
I wonder if it is possible to create a function that "completes" the work for both subnet managers. For kube, it would set the networkunavailable condition to False and for etcd, it would start monitoring the leases. With the current PR, we would have both functions in main.go which I think is not the best approach :(. Besides, we have functions here that are implemented by only one subnetManager (SetNodeNetworkUnavailableFalse for kube and RenewLease or WatchLease for etcd). If we could move all these to a general function under the Manager called something like "CompleteNetworkConfig", that would look much better. Could you spend some time on this? We could help

Reply:
I create other function CompleteLease which  for kube, it would set the networkunavailable condition to False and for etcd, it would start monitoring the leases. look at new MR